### ISSUE TITLE
Add exclude to meter matching, based on driver, mfct, type or ver

### ISSUE THREAD

**Issue by MaxPeal** [weetmuts/wmbusmeters#420]
As a User it would be nice feature to exclude meter devices form the meter matching.
like https://github.com/weetmuts/wmbusmeters/issues/29
but also with based form the driver, mfct, type or ver.

My scenario: Form my location, I receive over 400 different meters!
but until now i haven't find all My meters, 
and also not all with missing meter detection or missing drivers.

**Comment by weetmuts** [comment]
Yes, the time has come to add more explicit selection. In most cases just knowing the id is enough.
Can you propose the command line and config file format for setting these values?
